src/lib/swapper/swappers/ThorchainSwapper/utils/thorService.ts

```diff
@@ -8,6 +8,8 @@ const cachedUrls = [
   '/lcd/thorchain/pools',
   '/lcd/thorchain/inbound_addresses',
   '/lcd/thorchain/pool/',
+  '/v2/pools',
+  '/v2/pool/',
```

Comment:
This is currently borked

<img width="1696" alt="image" src="https://github.com/shapeshift/web/assets/17035424/239fb3df-4bf4-4a9a-9234-d1b04ae3f8d4">

`REACT_APP_MIDGARD_URL` already includes `/v2` path